backend/zulu/routes.py

```diff
@@ -34,12 +38,32 @@ def create_point(location: UserLocation, db=Depends(points_db)):
         "story": {
             'title': location.story.title,
             'content': location.story.content,
+            'image_id': location.story.image_id,
         },
         "user_id": location.user_id
     })
     return {}
 
 
+@api.get("/image")
+def get_image(id: str, db=Depends(images_db)):
+    try:
+        record = db.find_one({'_id': ObjectId(id)})
+    except InvalidId:
+        raise HTTPException(status_code=404, detail="Image not found. Id is not valid")
+    if not record:
+        raise HTTPException(status_code=404, detail="Image not found")
+    return StreamingResponse(io.BytesIO(record['image']), media_type=record['content_type'])
+
+
+@api.post("/image", status_code=status.HTTP_201_CREATED)
+def add_image(image: UploadFile = File(...), db=Depends(images_db)):
+    image_id = db.insert({'image': image.file.read(),
+                          'content_type': image.content_type,
+                          'filename': image.filename})
+    return {'id': str(image_id), 'filename': image.filename}
```

Comment:
1. Also not super-urgent - but can you add a schema for it? should be the ResponseModel
2. I am not sure if we need the filename - I think it depends on the browser implementation what would be the name.
3. One last thing - images should probably be stored as bytes and not string

Reply:
1. Just added.
2. Right. It is only for testing. Can be removed if disturb.
3. Yes they are. Why do you think they are not?